example/loaders/load_forced.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+from wasabi import msg
+
+from psycopmlutils.loaders.sql_load import sql_load
+from psycopmlutils.utils import data_loaders
+
+
+class LoadForce:
+    @data_loaders.register("forced_admissions_indlagt")
+    def forced_admissions_indlagt():
+
```

Comment:
```suggestion
    def forced_admissions():
```

An admission is an "indlæggelse" ❤️:
> the act or process of accepting someone into a hospital, clinic, or other treatment facility as an inpatient